src/routes/popup/tier.tsx

```diff
@@ -39,12 +40,23 @@ export function TierView() {
   const { data: activeTier } = useActiveTier();
 
   const formattedBalance = useMemo(() => {
+    if (!activeTier) return { displayBalance: "0", tooltipBalance: "0", showTooltip: false };
     const fractionedBalance = balanceToFractioned(String(activeTier?.balance ?? 0), {
       decimals: wanderTokenInfo.Denomination,
     });
     return formatBalance(fractionedBalance);
```

Comment:
You can play around with `fractionedBalance` value to check if the balance is not overflowing. It usually happens with very low balances like less than 0.